modules/statics/src/coins.ts

```diff
@@ -108,6 +109,7 @@ export const coins = CoinMap.fromCoins([
   account('txtz', 'Testnet Tezos', Networks.test.xtz, 6, UnderlyingAsset.XTZ, XTZ_FEATURES),
   account('susd', 'Silvergate USD', Networks.main.susd, 2, UnderlyingAsset.USD),
   account('tsusd', 'Testnet Silvergate USD', Networks.test.susd, 2, UnderlyingAsset.USD),
+  account('tfiat', 'Testnet FIAT', Networks.test.fiat, 2, UnderlyingAsset.FIAT),
```

Comment:
does this make sense as an account coin? What's the distinction between a fiat coin that uses the `account()` function vs one that uses the `tfiatToken()` function?

Reply:
Right, we aren't going to use Fiat or TFiat as coin, we will be creating just Fiat Tokens, so this call was removed as well.